### PR TITLE
Fix issues with output text and ig pkg name

### PR DESCRIPTION
**Pull request by ThishaniLucas**
Fixing the following:
https://github.com/wso2-enterprise/open-healthcare/issues/1441
https://github.com/wso2-enterprise/open-healthcare/issues/1442

### CODE REVIEW

File: native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/model/FHIRProfile.java [hunk 1]
```diff
@@ -130,6 +130,12 @@ public String getPackagePrefix() {
     }
 
     public void setPackagePrefix(BallerinaProjectToolConfig config) {
-        this.packagePrefix = config.getResourcePackage().substring(config.getResourcePackage().lastIndexOf(".") + 1);
+        String resourcePkg = config.getResourcePackage();
+        if (resourcePkg.indexOf('/') != -1) {
```
Comment (sameeragunarathne) [inline native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/model/FHIRProfile.java:134]: shall we use contains()?

File: native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/model/FHIRProfile.java [hunk 2]
```diff
@@ -130,6 +130,12 @@ public String getPackagePrefix() {
     }
 
     public void setPackagePrefix(BallerinaProjectToolConfig config) {
-        this.packagePrefix = config.getResourcePackage().substring(config.getResourcePackage().lastIndexOf(".") + 1);
+        String resourcePkg = config.getResourcePackage();
```
Comment (sameeragunarathne) [inline native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/model/FHIRProfile.java:133]: We have renamed the command arg to ig-package? if so shall we change the var to igPackage as well?

File: native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java [hunk 3]
```diff
@@ -143,8 +143,11 @@ public void execute() {
             HealthCmdUtils.exitError(exitWhenFinish);
         }
         if (this.engageSubCommand(argList)) {
-            printStream.println("Ballerina FHIR package generation completed successfully. Generated " +
-                    mode + " can be found at " + targetOutputPath);
+            if ("template".equals(mode)) {
```
Comment (sameeragunarathne) [inline native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java:146]: Let's use already available constant

File: native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/model/FHIRProfile.java (hunk 1 above)
Comment (ThishaniLucas) [reply native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/model/FHIRProfile.java]: fixed

File: native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/model/FHIRProfile.java (hunk 2 above)
Comment (ThishaniLucas) [reply native/fhir-to-bal-template/src/main/java/org/wso2/healthcare/fhir/codegen/ballerina/project/tool/model/FHIRProfile.java]: done

File: native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java (hunk 3 above)
Comment (ThishaniLucas) [reply native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java]: done

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**